theme_bookstore/views/snippets/s_showcase.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_showcase" inherit_id="website.s_showcase">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="pt80 pb80" remove="pt48 pb48"/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="//h3" position="replace" mode="inner">
+        We also sell prime quality second hand books
+    </xpath>
+    <!-- Lead -->
+    <xpath expr="//p[hasclass('lead')]" position="replace" mode="inner">
+        Prime quality second-hand books, carefully selected for value and charm— timeless stories at great prices.
```

Comment:
Is the "charm-" made like "I was about to say charming but i preferred to say timeless instead" or is there a typo ?